document/core/exec/runtime.rst

```diff
@@ -752,11 +752,11 @@ Throw contexts allow matching the program context around a throw instruction up
       \end{array}
 
 
+   When a throw of the form :math:`val^m (\THROWadm~a)` occurs, search for the maximal surrounding throw context :math:`T` is performed,
+   which means any other values, labels, frames, and |CAUGHTadm| instructions surrounding the throw :math:`val^m (\THROWadm~a)` are popped,
+   until an exception handler (corresponding to a try block) that :ref:`handles the exception <syntax-handler>` is found.
+   Then the values :math:`val^m:[t^m]` are appended to the tag address :math:`a` into a new |CAUGHTadm| instruction, which is pushed onto the stack.
```

Comment:
I mechanically converted sentences into passive forms to avoid 'we', but I'm not very sure what this sentence means. What does "values are appended to the tag address" mean?

Reply:
> What does "values are appended to the tag address" mean?

I meant this as in the "list" `a val^m` starting with the exception tag's address (`a`) and the rest is the list of values the exception carries (`val^m`). This "list" is contained in the instruction `caught{a val^m}`. 

I see how the verb "appended" is confusing or misleading here, that this is a list is irrelevant. I am rephrasing to:

> Then a new |CAUGHTadm|, containing the tag address :math:`a` and the values :math;`\val^m`, is pushed onto the stack.

Thanks!

Reply:
```suggestion
   Then a new |CAUGHTadm| instruction, containing the tag address :math:`a` and the values :math:`\val^m`, is pushed onto the stack.
```